woocommerce/woocommerce-admin
# add DevDocs for ScrollTo component

Fixes https://github.com/woocommerce/woocommerce-admin/issues/2947
See #2243 

Adds the DevDocs for the `<ScrollTo />` component added in #2799 .

### Detailed test instructions:

- Go to WooCommerce -> DevDocs
- Scroll down to `ScrollTo` component
- Click and check the detailed documentation

### Changelog Note:

None needed.
